explorer_frontend/.env.example

```diff
@@ -0,0 +1,7 @@
+# Application Metadata
+VITE_APP_TITLE=
```

Comment:
Not needed. We are using runtime configuration for that app. It gallows to build it once and then adjust or change any parameters we need without a rebuild.

Reply:
Will remove!